cypress/integration/inventory/Item-status-date-updates.spec.js

```diff
@@ -0,0 +1,440 @@
+            cy.getOrganizationApi({ query: 'name="Amazon.com"' })
+              .then(organization => {
+                order.vendor = organization.id;
+                orderLine.physical.materialSupplier = organization.id;
+                orderLine.eresource.accessProvider = organization.id;
+              });
+            cy.getMaterialTypes({ query: 'name="book"' })
+              .then(materialType => { orderLine.physical.materialType = materialType.id; });
+          });
+        ServicePoints.createViaApi(effectiveLocationServicePoint.body);
+        ServicePoints.createViaApi(notEffectiveLocationServicePoint.body);
+        cy.addServicePointToUser([effectiveLocationServicePoint.body.id, notEffectiveLocationServicePoint.body.id],
+          user.userId, effectiveLocationServicePoint.body.id);
+        Locations.createLocationViaApi(effectiveLocation)
+          .then(locations => {
+            orderLine.locations[0].locationId = locations.body.id;
+            itemLocation = locations.body.name;
+          });
+        cy.login(userProperties.username, userProperties.password)
+          .then(() => {
+            Orders.createOrderWithOrderLineViaApi(order, orderLine)
+              .then(orderNumber => {
+                createdOrderNumber = orderNumber;
+              });
+          })
+          .then(() => {
+            cy.getRequestPolicy({ query: `name=="${REQUEST_POLICY_NAMES.ALLOW_ALL}"` });
+            cy.getNoticePolicy({ query: `name=="${NOTICE_POLICY_NAMES.SEND_NO_NOTICES}"` });
+            cy.getOverdueFinePolicy({ query: `name=="${OVERDUE_FINE_POLICY_NAMES.OVERDUE_FINE_POLICY}"` });
+            cy.getLostItemFeesPolicy({ query: `name=="${LOST_ITEM_FEES_POLICY_NAMES.LOST_ITEM_FEES_POLICY}"` });
+            cy.getLoanPolicy({ query: `name=="${LOAN_POLICY_NAMES.EXAMPLE_LOAN_POLICY}"` });
+            cy.getCirculationRules()
+              .then(rules => {
+                rulesDefaultString = rules.rulesAsText;
+              });
+          })
+          .then(() => {
+            const requestPolicyId = Cypress.env(CY_ENV.REQUEST_POLICY)[0].id;
+            const noticePolicyId = Cypress.env(CY_ENV.NOTICE_POLICY)[0].id;
+            const overdueFinePolicyId = Cypress.env(CY_ENV.OVERDUE_FINE_POLICY)[0].id;
+            const lostItemFeesPolicyId = Cypress.env(CY_ENV.LOST_ITEM_FEES_POLICY)[0].id;
+            const loanPolicyId = Cypress.env(CY_ENV.LOAN_POLICIES)[0].id;
+            const newRule = `\ng ${user.patronGroup} + m ${orderLine.physical.materialType}: l ${loanPolicyId} r ${requestPolicyId} n ${noticePolicyId} o ${overdueFinePolicyId} i ${lostItemFeesPolicyId}`;
+
+            cy.updateCirculationRules({
+              rulesAsText: rulesDefaultString + newRule,
+            });
+          });
+        cy.getUsers({ limit: 1, query: `"personal.lastName"="${user.username}" and "active"="true"` })
+          .then((users) => {
+            userBarcode = users[0].barcode;
+          });
+      });
```

Comment:
Move 131-189 strings in functions inside fragments with undertandable names

Reply:
deleted some piece of code